token/cli/src/output.rs

```diff
@@ -399,6 +481,108 @@ impl fmt::Display for CliTokenAccounts {
     }
 }
 
+fn display_ui_extension(
+    f: &mut fmt::Formatter,
+    epoch: u64,
+    ui_extension: &UiExtension,
+) -> fmt::Result {
+    match ui_extension {
+        UiExtension::TransferFeeConfig(UiTransferFeeConfig {
+            transfer_fee_config_authority,
+            withdraw_withheld_authority,
+            withheld_amount,
+            older_transfer_fee,
+            newer_transfer_fee,
+            ..
+        }) => {
+            let UiTransferFee {
+                transfer_fee_basis_points,
+                maximum_fee,
+                ..
+            } = if newer_transfer_fee.epoch >= epoch {
+                newer_transfer_fee
+            } else {
+                older_transfer_fee
+            };
+
+            writeln!(f, "  {}", style("Transfer fees:").bold())?;
+            writeln!(
+                f,
+                "    {} {}bps",
+                style("Fee:").bold(),
+                transfer_fee_basis_points
+            )?;
+            writeln_name_value(f, "    Maximum fee:", &maximum_fee.to_string())?;
+            writeln_name_value(
+                f,
+                "    Config authority:",
+                transfer_fee_config_authority
+                    .as_ref()
+                    .unwrap_or(&String::new()),
+            )?;
+            writeln_name_value(
+                f,
+                "    Withdrawal authority:",
+                withdraw_withheld_authority
+                    .as_ref()
+                    .unwrap_or(&String::new()),
+            )?;
+            writeln_name_value(f, "    Withheld fees:", &withheld_amount.to_string())
+        }
+        UiExtension::TransferFeeAmount(UiTransferFeeAmount { withheld_amount }) => {
+            writeln_name_value(f, "  Transfer fees withheld:", &withheld_amount.to_string())
+        }
+        UiExtension::MintCloseAuthority(UiMintCloseAuthority { close_authority }) => {
+            writeln_name_value(
+                f,
+                "  Close authority:",
+                close_authority.as_ref().unwrap_or(&String::new()),
+            )
+        }
+        UiExtension::ConfidentialTransferMint(_) => unimplemented!(),
+        UiExtension::ConfidentialTransferAccount(_) => unimplemented!(),
```

Comment:
i think im going to punt this for now because the confidential code is complex and im not sure what we want to do for like, possibly decrypting and displaying values

Reply:
@samkim-crypto will probably have some nice ideas :smile:

Reply:
Yep, the confidential part requires key derivation and decryption, so it should probably be tackled in a separate PR. I do have ideas on displaying it, so I can build on this later. Thanks!